### PR TITLE
feat(node-analyzer): host-scanner bump to 0.3.4

### PR DESCRIPTION
**Pull request by dippie8**
## Checklist

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [ ] Title of the PR starts with type and scope, (e.g. feat(agent,node-analyzer,sysdig-deploy):)
- [ ] Chart Version bumped for the respective charts
- [ ] Variables are documented in the README.md (or README.tpl in some charts)
- [ ] Check GithubAction checks (like lint) to avoid merge-check stoppers
- [ ] All test files are added in the tests folder of their respective chart and have a "_test" suffix

Check Contribution guidelines in README.md for more insight.

### CODE REVIEW

File: charts/node-analyzer/values.yaml [hunk 1]
```diff
@@ -334,6 +334,8 @@ nodeAnalyzer:
         memory: 150Mi
         ephemeral-storage: 250Mi
 
+    probesPort: 7001
```
Comment (ProvoK) [inline charts/node-analyzer/values.yaml]: I would move this before `image:` for better readability 🙏

File: charts/node-analyzer/README.md [hunk 2]
```diff
@@ -154,7 +154,7 @@ The following table lists the configurable parameters of the Sysdig Node Analyze
...
-| `nodeAnalyzer.hostScanner.image.tag`                                 | The image tag to pull the Host Scanner                                                   | `0.3.2`                                                                                                                                                               |
+| `nodeAnalyzer.hostScanner.image.tag`                                 | The image tag to pull the Host Scanner                                                   | `0.3.4`                                                                                                                                                               |
 | `nodeAnalyzer.hostScanner.image.digest`                              | The image digest to pull                                                                 | ` `                                                                                                                                                                   |
```
Comment (michele-mangili) [inline charts/node-analyzer/README.md:158]: Should we add to the readme the `nodeAnalyzer.hostScanner.probesPort` parameter?

Comment (ProvoK) [reply charts/node-analyzer/README.md:158]: yes please

File: charts/node-analyzer/values.yaml [hunk 3]
```diff
@@ -318,10 +318,11 @@ nodeAnalyzer:
     # additionalDirsToScan is a optional comma-separated list of directories that
     # should be analyzer in addition to default ones.
     # additionalDirsToScan: "/foo/bar/baz,/my/other/folder"
+    probesPort: 7001
```
Comment (ProvoK) [inline charts/node-analyzer/values.yaml:323]: please @dippie8 add a comment here (basically the README's one)
```suggestion

    // probesPort ...
    probesPort: 7001
```

**Review by michele-mangili — APPROVED** [review]

**Review by ProvoK — APPROVED** [review]

**Pull request merged**